tests/testthat/helper-lazy.R

```diff
@@ -0,0 +1,3 @@
+verify_lazy_output <- function(...) {
```

Comment:
Rather than creating a new function, I would recommend using `verify_output()` + `local_options(dplyr_table_num = 0)`

Reply:
`local_options()` doesn't work as expected in `verify_output()`, the option, the helper seems to be the easiest way. Do you prefer a different name?

``` r
testthat::verify_output("foo.txt", {
  rlang::local_options(foo = 1)
  getOption("foo")
})
#> Warning: Creating reference output

cat(readLines("foo.txt"), sep = "\n")
#> > rlang::local_options(foo = 1)
#> > getOption("foo")
#> NULL
```

<sup>Created on 2020-05-19 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Reply:
The `local_options()` would go outside the `verify_output()`:

``` r
local({
  rlang::local_options(foo = 1)
  testthat::verify_output("foo.txt", {
    getOption("foo")
  })
})
#> Warning: Creating reference output
cat(readLines("foo.txt"), sep = "\n")
#> > getOption("foo")
#> [1] 1
```

<sup>Created on 2020-05-19 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>